packages/common-integration-tests/src/index.ts

```diff
@@ -1,16 +1,17 @@
 export * from './common-int-test-utils';
-export * from './create-delete-list-cache';
-export * from './get-set-delete';
-export * from './auth-client';
-export * from './list';
-export * from './dictionary';
-export * from './set';
-export * from './sorted-set';
-export * from './item-get-type';
-export * from './topic-client';
-export * from './keys-exist';
-export * from './update-ttl';
-export * from './leaderboard-client';
-export * from './webhooks';
-export * from './batch-get-set';
-export * from './storage';
+export * from './cache/create-delete-list-cache';
+export * from './cache/get-set-delete';
+export * from './auth/auth-client';
+export * from './cache/list';
+export * from './cache/dictionary';
+export * from './cache/set';
+export * from './cache/sorted-set';
+export * from './cache/item-get-ttl';
+export * from './cache/item-get-type';
+export * from './topics/topic-client';
+export * from './cache/keys-exist';
+export * from './cache/update-ttl';
+export * from './leaderboard/leaderboard-client';
+export * from './topics/webhooks';
+export * from './cache/batch-get-set';
+export * from './storage/storage';
```

Comment:
In a future PR, we could export the service names only as opposed to all the names within each of the modules. That way we can properly namespace them in tests. Since this is an internal refactor, we leave that for future work.